### ISSUE TITLE
Use SpeechRecognition and audioCapture simultaneously

### ISSUE THREAD

**Issue by desmeit** [pbakondy/cordova-plugin-speechrecognition#82]
Hello,

is it possible to use your plugin and this plugin

https://github.com/edimuj/cordova-plugin-audioinput

simultaneously? I want to try the following in this direction:

// audioinput plugin
audioinput.start();

// speechrecognition plugin
window.plugins.speechRecognition.startListening(function(result){}

// speechrecognition plugin
window.plugins.speechRecognition.stopListening(function(result){}

// audioinput plugin
audioinput.stop();

Is this working?

**Comment by bknill** [comment]
Did you figure out how to get this working?

**Comment by desmeit** [comment]
Unfortunately not. Is there any support for this plugin? Nobody is answering here.

**Comment by AugusDogus** [comment]
@desmeit Not really. If you find a bug and make a PR it should come down the pipe, but I imagine @pbakondy made this for an app that he built and it worked fine for his purposes and then he moved onto other things. If you're using this with Ionic, they should be releasing a set of their own plugins through capacitor sometime down the line, but for now it's really up to us/you to fix any issues you're having.

**Comment by Marcophono2** [comment]
Any news here?
